packages/commercetools/composables/src/useCategory/index.ts

```diff
@@ -1,11 +1,11 @@
-import { UseCategory } from '@vue-storefront/core';
+import { CustomQuery, UseCategory } from '@vue-storefront/core';
 import { getCategory } from '@vue-storefront/commercetools-api';
 import { Category } from './../types/GraphQL';
 import { useCategoryFactory, UseCategoryFactoryParams } from '@vue-storefront/core';
 
 const params: UseCategoryFactoryParams<Category, any> = {
-  categorySearch: async (params) => {
-    const categoryResponse = await getCategory(params);
+  categorySearch: async (params, customQuery: CustomQuery) => {
```

Comment:
`customQuery` should be optional, same for product

Reply:
Done.